pages/proposals/language/Espanol.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Yea I will need to make final edits to remove references to the `/proposals/espanol` page before we push these live.

@TheoChevalier should we give Spanish a deadline? We can also add it in later. I just wanna make sure German and French pages go live on time.